modules/quadbin/redshift/sql/QUADBIN_FROMZXY.sql

```diff
@@ -3,7 +3,7 @@
 ----------------------------
 
 CREATE OR REPLACE FUNCTION @@RS_PREFIX@@carto.QUADBIN_FROMZXY
-(z INT, x INT, y INT)
+(z BIGINT, x BIGINT, y BIGINT)
```

Comment:
I would keep INT. Is there any reason to update to BIGINT?

Reply:
As discussed in Slack the UDF QUADBIN_FROMZXY was complaining when we use columns in a table of type BIGINT. The customers would have to explicitly cast to int. This way we avoid those problems.